lucida/imagematching/opencv_imm/server/Image.cpp

```diff
@@ -31,220 +31,223 @@ namespace po = boost::program_options;
 namespace fs = boost::filesystem;
 
 string Image::saveToFS(const string &data) {
-	struct timeval tp;
-	gettimeofday(&tp, NULL);
-	long int timestamp = tp.tv_sec * 1000 + tp.tv_usec / 1000;
-	string image_path = "input-" + to_string(timestamp) + ".jpg";
-	ofstream image_file(image_path, ios::binary);
-	print("Size of image written to FS: " << data.size());
-	image_file.write(data.c_str(), data.size());
-	image_file.close();
-	return image_path;
+   struct timeval tp;
```

Comment:
Yeah, the actual addition is on line 119 to convert images before pushing them to DB